config/config.yaml

```
@@ -124,4 +124,4 @@ params:
   #     false positive adapter matches
   #   * `--minimum-overlap 7`: the cutadapt default minimum overlap of `5` did trimming on the level
   #     of expected adapter matches by chance
-  cutadapt-pe: "-a ACGGATCGATCGATCGATCGAT -g GGATCGATCGATCGATCGAT -A ACGGATCGATCGATCGATCGAT -G GGATCGATCGATCGATCGAT --minimum-length 33 -e 0.005 --overlap 7"
+  cutadapt-pe: "-j 0 -a ACGGATCGATCGATCGATCGAT -g GGATCGATCGATCGATCGAT -A ACGGATCGATCGATCGATCGAT -G GGATCGATCGATCGATCGAT --minimum-length 33 -e 0.005 --overlap 7"
```

You should not have to specify `-j` or `--cores` explicitly, as the wrappers take care of passing on the `threads: ` specification. See the [respective wrapper code for the `pe` setup](https://github.com/snakemake/snakemake-wrappers/blob/4ec0c91f37e037479c40551463449d96830a548e/bio/cutadapt/pe/wrapper.py#L25) and the [respective wrapper code for the `se` setup](https://github.com/snakemake/snakemake-wrappers/blob/4ec0c91f37e037479c40551463449d96830a548e/bio/cutadapt/se/wrapper.py#L25).

Thus, I suggest to remove this change:
```suggestion
  cutadapt-pe: "-a ACGGATCGATCGATCGATCGAT -g GGATCGATCGATCGATCGAT -A ACGGATCGATCGATCGATCGAT -G GGATCGATCGATCGATCGAT --minimum-length 33 -e 0.005 --overlap 7"
```